tracks/ruby/exercises/leap/readme.md

```diff
@@ -0,0 +1,30 @@
+_Leap_ is one of the first side exercises, unlocked by _TwoFer_. 
+
+## Reasonable Solutions
+
+```ruby
+class Leap
+  year % 4 == 0 && (year % 100 != 0 || year % 400 == 0)
+end
+```
+Variants: `module` instead of `class`; instantiate class.  
+
+
+## Common suggestions
+Copied from the Python notes by @yawpitch :
```

Comment:
- It took me a long time to find the source when I went looking; this is an easy reference.
- I'd rather not have `git blame` finding me, as if I came up with this myself
(- There's been some talk about adding track agnostic notes for exercises; this will be easy to find as a candidate.)
So I'd rather leave it in. Okay?

Reply:
I don't mind you leaving it in, but I'll PR it out straight afterwards :)

Reply:
> There's been some talk about adding track agnostic notes for exercises; this will be easy to find as a candidate.

I personally don't see this working, as exercises are aiming to teach the specific idioms of a language rather than programming concepts in general.

Reply:
> I don't mind you leaving it in, but I'll PR it out straight afterwards :)

Aha, great solution. That solves both of my problems :-)